eo-maven-plugin/src/main/resources/org/eolang/maven/add_rust/add_rust.xsl

```diff
@@ -37,6 +37,17 @@ SOFTWARE.
               <xsl:attribute name="loc">
                 <xsl:value-of select="../attribute(loc)"/>
               </xsl:attribute>
+              <dependencies>
+                <xsl:for-each select="../o/o">
```

Comment:
@levBagryansky I believe searching the parent node attribute has separate value. Something like this:
```
<xsl:for-each select="ancestor::o[1]/@o">
```